src/devices/HuskyLens/samples/HuskyLens.Samples.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\HuskyLens.csproj" />
+    <PackageReference Include="System.IO.Ports" Version="4.7.0" />
```

Comment:
I think this should be like this: <PackageReference Include="System.IO.Ports" Version="$(SystemIOPortsPackageVersion)" />